### ISSUE TITLE
Feedback for “Quickstart”

### ISSUE THREAD

**Issue by agamm** [langfuse/langfuse-docs#602]
The imports aren't there in the code example for "any LLM". Also the langfuse_context is undefined.

**Comment by marcklingen** [comment]
thanks for the feedback!

**Comment by agamm** [comment]
BTW, will it track the output (and costs related to the io?)

**Comment by marcklingen** [comment]
> BTW, will it track the output (and costs related to the io?)

yes, by default logs the output and you can set costs manually or use Langfuse for tokenization
